examples/test/utils/helpers.js

```diff
@@ -2,22 +2,34 @@
 import { t as testController, Selector } from 'testcafe';
 import { config } from './config';
 
-export async function checkCellValue(t, selector, cellValue, cellSelector = '.ag-cell') {
+async function getCell(t, selector, cellSelector) {
     const chart = Selector(selector);
-    await t
-        .expect(chart.exists).eql(true, `${selector} not found`);
-    if (cellSelector) {
-        const cell = chart.find(cellSelector);
-        await t
-            .expect(cell.exists).eql(true, `${cellSelector} not found in ${selector}`);
+    await t.expect(chart.exists).eql(true, `${selector} not found`);
 
-        if (cellValue) {
-            await t
-                .expect(cell.textContent).eql(cellValue, `expected ${cellSelector} to contain text ${cellValue}`);
-        }
+    const cell = await chart.find(cellSelector);
+    await t.expect(cell.exists).eql(true, `${cellSelector} not found in ${selector}`);
+
+    return cell;
+}
+
+export async function checkCellValue(t, selector, cellValue, cellSelector = '.ag-cell') {
+    const cell = await getCell(t, selector, cellSelector);
+    if (cellValue) {
+        await t
+            .expect(cell.textContent).eql(cellValue, `expected ${cellSelector} to contain text ${cellValue}`);
     }
 }
 
+export async function checkCellHasClassName(t, selector, expectedClassName, cellSelector) {
+    const cell = await getCell(t, selector, cellSelector);
+    await t.expect(cell.hasClass(expectedClassName)).ok(`expected ${cellSelector} to has class ${expectedClassName}`);
```

Comment:
I know its ok to use `ok` here but with my experience other developers could miss-use this functionality by copy&paste :/ so I would recommend to use `eql` checks to avoid this situations.

Reply:
Ok, thanks for info.